CHANGELOG.md

```diff
@@ -15,3 +15,7 @@
 - Fix some bugs
 - Add edit --classifcation  to reclassify an asset
 - Add the save asset command
+
+## March 25th,2024
+
+- add new cmd for  CLI
```

Comment:
Can you please here mention it is the "clear" command other than that it looks good

Reply:
hey i added  it 
can you look over